exporters/prometheus/exporter_test.go

```diff
@@ -262,7 +300,10 @@ func TestPrometheusExporter(t *testing.T) {
 				metric.WithResource(res),
 				metric.WithReader(exporter, customBucketsView, defaultView),
 			)
-			meter := provider.Meter("testmeter")
+			meter := provider.Meter(
+				"testmeter",
+				otelmetric.WithInstrumentationVersion("v0.1.0"),
+			)
```

Comment:
Can you add another test that validates multiple scopes from different meters are assigned correctly to the instruments they create?

Reply:
This looks unresolved. @fatsheep9146 are you still planning to address this or did I miss the addition that already did?

Reply:
The test is added, `TestMultiScopes`.